docs/style-guide.md

```diff
@@ -0,0 +1,80 @@
+# Project-Base Challenge Style Guide
+
+This guide aims to explain our overall approach to how project-based challenges (e.g. Step 1) are structured and written.
+
+## Instructional Content
+
+The instructions for a step challenge should be as concise as possible to inform the camper what to do, with the exception of steps that introduce a new concept. When introducing a new concept, you'll generally use the following format:
+
+```md
+Definition
+
+Explanation
+
+Instructions
+```
+
+As an example:
```

Comment:
[proofreading] Contributors might wonder if the sentence is going to introduce the example or if it is going to describe what the example is doing. If it's the former, check out the suggestion below to help you revise it:  
```suggestion
Here is an example:
```